packages/browser/src/index.ts

```diff
@@ -48,12 +48,14 @@ export default class LogtoClient {
   protected logtoConfig: LogtoConfig;
   protected oidcConfig?: OidcConfigResponse;
   protected logtoStorageKey: string;
+  protected requester: Requester;
 
-  constructor(logtoConfig: LogtoConfig) {
+  constructor(logtoConfig: LogtoConfig, requester = createRequester()) {
     this.logtoConfig = logtoConfig;
     this.logtoStorageKey = getLogtoKey(logtoConfig.clientId);
-    this.idToken = conditional(localStorage.getItem(`${this.logtoStorageKey}:idToken`));
+    this.requester = requester;
     this.refreshToken = conditional(localStorage.getItem(`${this.logtoStorageKey}:refreshToken`));
+    this.idToken = conditional(localStorage.getItem(`${this.logtoStorageKey}:idToken`));
```

Comment:
should be nullable instead of undefined?

Reply:
just intended to sort the statements in the same order of the related object properties here.

```ts
export default class LogtoClient {
  ……
  protected refreshToken?: string;
  protected idToken?: string;
```

we can do nullable later.
LOG-1576